### ISSUE TITLE
R 4.1: POST request with Content-Type "application/json" fails to process parameters with null values 

### ISSUE THREAD

**Issue by RomanShevtsiv** [opencpu/opencpu#389]
I'm using R 4.1.2, openCPU 2.2.5.

I have a simple test function inside a simple test package:
```
test_json(param_a, param_b) <- {
    response <- list(result = "OK")
    return(response)
}
```

I run opencpu localy `ocpu_start_server()` and test it using postman request: 
```
{
  "param_a": "test",
  "param_b": null
}
```

Getting the following error:

> attempt to set an attribute on NULL
> In call:
> class(x) <- unique.default(c("AsIs", oldClass(x)))

The above works fine with R 4.0 and several versions below.
If I replace null with any empty object like [] or {} or "" everything is OK.
I've been using openCPU for the last 5 years and never faced anything similar before.

**Comment by stefvanbuuren** [comment]
Did you perhaps mean `
test_json <- function(param_a, param_b){ response <- list(result = "OK"); return(response) }`?

**Comment by RomanShevtsiv** [comment]
> Did you perhaps mean ` test_json <- function(param_a, param_b){ response <- list(result = "OK"); return(response) }`?

Yes. It's actually a formatting issue. Should be:
```
test_json <- function(param_a, param_b){ 
    response <- list(result = "OK")
    return(response)
}
```

**Comment by jeroen** [comment]
Thanks I can reproduce this. Seems like a change in R 4.1

```sh
curl https://cloud.opencpu.org/ocpu/library/base/R/list/print -H "Content-Type: application/json" -d '{"x":123,"y":null}'
```

**Comment by jeroen** [comment]
OK this should be fixed now. Good catch :)

**Comment by jeroen** [comment]
Released in 2.2.6
